### ISSUE TITLE
Question about training procedure

### ISSUE THREAD

**Issue by StanislasBertrand** [euagendas/m3inference#1]
Hi, 
First of all thank you for your great work.

I was wondering what you used as ground truth label for age and gender when user profiles are organizations. You wouldn't want the model to train to recognize any gender / age on an organization profile. 
I believe this is not mentioned in the article, or maybe I misunderstood something about the training procedure ?

**Comment by davidjurgens** [comment]
Organizational accounts are excluded from the training sets for age and gender.  Each of the tasks (age/gender/is-org) use separate training sets, as not all accounts have all the labels.

**Comment by StanislasBertrand** [comment]
This makes sense for the pre-training of the separate models, but for the fine-tuning of the entire M3 model, wouldn't you need input users that have a ground truth for every attribute ? Maybe you could define a loss function that only calculates the error on organization (cutting out softmax for age / gender) when an organization user is used as training input ?

**Comment by davidjurgens** [comment]
The fused models are fine-tuned using one attribute at a time.  (Single-task fine-tuning for multi-modal input)  I don't think we looked into a joint loss because so few accounts have both age and gender.

**Comment by StanislasBertrand** [comment]
Ok, thank you for clarifying.
